src/vcpkg/portfileprovider.cpp

```diff
@@ -88,11 +88,12 @@ namespace vcpkg::PortFileProvider
         return Util::fmap(m, [](const auto& p) { return p.second; });
     }
 
-    DECLARE_AND_REGISTER_MESSAGE(VersionSpecMismatch,
-                                 (msg::path, msg::expected_version, msg::actual_version),
-                                 "",
-                                 "error: Failed to load port because version specs did not match\n    Path: "
-                                 "{path}\n    Expected: {expected_version}\n    Actual: {actual_version}");
+    DECLARE_AND_REGISTER_MESSAGE(
+        VersionSpecMismatch,
+        (msg::path, msg::expected_version, msg::actual_version),
+        "",
+        "Failed to load port because versions are inconsistent. The file \"{path}\" contains the version "
```

Comment:
I rewrote this error to make sense without the indent formatting. I could resolve it differently by breaking it up in to separate messages to preserve the indenting if people want.